homeassistant/components/switcher_kis/button.py

```diff
@@ -26,15 +23,14 @@
 
 PARALLEL_UPDATES = 1
 
+API_CONTROL_BREEZE_DEVICE = "control_breeze_device"
+
```

Comment:
I think this constant is now used in two modules (button and climate) and should be moved to `const.py`